tests/HTTP/ClientTest.php

```diff
@@ -209,12 +209,19 @@ public function testSendToGetLargeContent()
             $this->markTestSkipped('Set an environment value BASEURL to continue');
         }
 
+        // Allow the peak memory usage limit to be specified externally, if needed.
+        // When running this test in different environments it may be appropriate to set a different limit.
+        $maxPeakMemoryUsage = (int) \getenv('SABRE_HTTP_TEST_GET_LARGE_CONTENT_MAX_PEAK_MEMORY_USAGE');
+        if ($maxPeakMemoryUsage === false) {
```

Comment:
Since prev line is casted ro int, I guess it cannot be false anymore?

Reply:
I shouldn't make changes at the last moment!